sql/src/main/java/org/apache/druid/sql/calcite/planner/QueryHandler.java

```diff
@@ -374,17 +384,27 @@ protected PlannerResult planExplanation(
       log.error(jpe, "Encountered exception while serializing resources for explain output");
       resourcesString = null;
     }
+
+    String statementAttributesString;
+    try {
+      statementAttributesString = plannerContext.getJsonMapper().writeValueAsString(plannerContext.getStatementAttributes());
```

Comment:
should we write this as a named object, instead of an anonymous: for example

```
{
  "statementAttributes": {
    "statementKind": ...,
    "targetDataSource": ...  
  }
}
```

```
{
    "statementKind": ...,
    "targetDataSource": ...  
}
```